### PR TITLE
Update officers.md

### PR DESCRIPTION
**Pull request by awhicks**

### CODE REVIEW

File: officers.md [hunk 1]
```diff
@@ -9,26 +9,26 @@ permalink: /officers/
 ## <a name="Officers"></a>Officers<a href="#Officers"><i class="fa fa-link" aria-hidden="true"></i></a>
 
 
-### <a name="Danny">[Danny Manesh](https://maneshd.github.io/), President<a href="#Danny"><i class="fa fa-link" aria-hidden="true"></i></a>
+### <a name="Wesley">[Wesley Woo](https://wesmwoo.github.io), President<a href="#Wesley"><i class="fa fa-link" aria-hidden="true"></i></a>
 
 Danny has taken control from Alex.
```
Comment (wesmwoo) [inline officers.md:14]: Wesley has taken control from Danny? >:)

Comment (awhicks) [reply officers.md]: 👀

Comment (awhicks) [reply officers.md]: Also feel free to change the little blurbs (or offer that the new officers change their own)

**Review by maneshd — APPROVED** [review]
LGTM

**Pull request merged**